Buildscripts/gh-download-artifacts.py

```
@@ -0,0 +1,135 @@
+    headers = {
+        "Authorization": f"Bearer {token}",
+        "Accept": "application/vnd.github+json",
+        "X-GitHub-Api-Version": "2022-11-28",
+    }
+
+    out_dir.mkdir(parents=True, exist_ok=True)
+
+    url = f"{API_URL}/repos/{repo}/actions/runs/{run_id}/artifacts"
+    artifacts = []
+    while url:
+        resp = requests.get(url, headers=headers, params={"per_page": 100})
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)